core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy

```diff
@@ -251,6 +251,11 @@ class TomcatConfigurator {
    *
    * For disabling byteman configuration, it is possible to call method `revertAllConfiguration()`,
    * however be aware that it removes all configuration changes applied.
+   *
+   * Byteman instance example:
+   * new Byteman(new BytemanInstaller().prepareBytemanJar())
+   *      .script([bytemanScriptFile1.btm, bytemanScriptFile2.btm])
```

Comment:
If syntax correct, shouldn't be is smth. like?

`script([new File(bytemanScriptFile1.btm), new File('bytemanScriptFile2.btm')]]) ...`


Please provide compile-able examples.